### DISCUSSION TITLE
Enhancing CloudStack Management UI Accessibility with Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by apurv-1** [apache/cloudstack#8955] [Ideas]
Hey folks,

We are planning to add keyboard shortcuts to CloudStack UI, I was wondering what are some of the most frequently used actions that user's perform on the UI. 

It would be great if all of us can list down our most frequent used actions? 
eg: 
1. Go to `Compute/Storage/Network/...` tab.
2. Add a `Instance/Network/...`.
3. etc.

Looking forward to hearing from you all!

**Comment by yadvr** [comment]
Good to hear @apurv-1 - after struggling to get your PR in, we abandoned our own effort mainly due to support with vue3 and latest ant-d. If you're up for the work, we are happy to review your work and accept it as long as it is functional.
cc @borisstoyanov
